### ISSUE TITLE
Enable-NetFirewallRule -DisplayGroup "Remote Administration"

### ISSUE THREAD

**Issue by eclaypool** [MicrosoftDocs/windowsserverdocs#821]
This FW group was removed starting with Windows Server 2012.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ebe4d150-8841-b5b8-4762-2383cb3a7163
* Version Independent ID: f57bcc2a-ca9b-eea2-f778-091d7678c6ed
* Content: [Manage Server Core](https://docs.microsoft.com/en-us/windows-server/administration/server-core/server-core-manage)
* Content Source: [WindowsServerDocs/administration/server-core/server-core-manage.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/server-core/server-core-manage.md)
* Service: **windows-server-threshold**
* GitHub Login: @lizap

**Comment by Ashish-smiley** [comment]
This is from Windows 2019.

So I don't think this group is removed.
PS C:\Users\Administrator> Get-NetFirewallRule -DisplayGroup "Windows Remote management" | fl Displaygroup


Displaygroup : Windows Remote Management

Displaygroup : Windows Remote Management

**Comment by mbtxau** [comment]
The post by @eclaypool is correct. The display group in this article "Remote Administration" was removed from Windows Server 2012 and beyond.  @Ashish-smiley the display group you have mentioned was "Windows Remote Management". 

As you can clearly see, these display groups have two completely different names, which is hugely important when dealing with command line or PowerShell commands. 

This article should be updated to remove the reference: 
Enable-NetFirewallRule -DisplayGroup "Remote Administration"  and then replace it with:
 Enable-NetFirewallRule -DisplayGroup "Windows Remote Management"

**Comment by Ashish-smiley** [comment]
That's why I put my comment so article can be updated with correct information.

Thanks,
Ashish

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by ArrowRaider** [comment]
The documentation is wrong here https://docs.microsoft.com/en-us/windows-server/administration/server-core/server-core-manage

**Comment by dennish78** [comment]
![2020-09-17 17_45_33-Window](https://user-images.githubusercontent.com/66772287/93494929-dc0dbb80-f90d-11ea-9ba0-c9ca40b8d1cf.png)
It's September 2020, almost 2 and a half years since this was reported. Why has this still not been fixed?
It's a 5 second change.

**Comment by illfated** [comment]
Sorry about the delay in finding out more about these details.
This issue has also been reported in issue ticket #4477 **Invalid Firewall command for Server 2019** .

I have been a bit reluctant to propose the change, seeing that neither of the cmdlet strings work on Windows Server 2016 LTS:

![image](https://user-images.githubusercontent.com/38162891/94994102-d4007d80-0595-11eb-9fa3-1a2dbf6044fc.png)

**Comment by illfated** [comment]
Would this be a more useful change to accommodate for all of the "Applies to:" versions (Windows Server 2019, Windows Server 2016, and Windows Server (Semi-Annual Channel))?

### To configure Windows Firewall to allow MMC snap-in(s) to connect
To allow all MMC snap-ins to connect, run the following command:

```PowerShell
Enable-NetFirewallRule
```

![Enable-NetFirewallRule](https://user-images.githubusercontent.com/38162891/95003022-fff22200-05da-11eb-85b1-13afdd6798c6.png)

To allow only specific MMC snap-ins to connect, run the following:

```PowerShell
Enable-NetFirewallRule -DisplayGroup "<rulegroup>"
```

Where *rulegroup* is one of the following, depending on which snap-in you want to connect:

| MMC snap-in                            | Rule group                                            |
| ---------------------------------------- | ------------------------------------------------------- |
| Event Viewer                           | Remote Event Log Management                           |
| Services                               | Remote Service Management                             |
| Shared Folders                         | File and Printer Sharing                              |
| Task Scheduler                         | Performance Logs and Alerts, File and Printer Sharing |
| Disk Management                        | Remote Volume Management                              |
| Windows Firewall and Advanced Security | Windows Firewall Remote Management                    |

**Comment by illfated** [comment]
Resolved in PR #5046 .

#please-close
